test/generate-plugin.test.js

```diff
@@ -100,16 +100,35 @@ function define (t) {
   })
 
   test('should finish succesfully', async (t) => {
-    t.plan(17 + Object.keys(expected).length)
+    t.plan(18 + Object.keys(expected).length)
     try {
       await generate(workdir, pluginTemplate)
       await verifyPkg(t)
       await verifyCopy(t, expected)
+      await verifyFastifyPluginVersion(t)
     } catch (err) {
       t.error(err)
     }
   })
 
+  function verifyFastifyPluginVersion (t) {
+    return new Promise((resolve, reject) => {
+      const pkgFile = path.join(workdir, 'package.json')
+
+      readFile(pkgFile, function (err, data) {
+        err && reject(err)
```

Comment:
With this syntax, the following code would be executed

```
if (err)
  return reject(err)
```

If you saw this style code here, we should fix it (in another PR)

Reply:
Yes, I see it from other tests in this file, such as verifyPkg. I could also change other code; no problem.